app/src/store/features/environment/types.ts

```diff
@@ -0,0 +1,10 @@
+import { EnvironmentVariable } from "backend/environment/types";
+
+export interface InitialState {
+  currentEnvironment: EnvironmentDetails;
+  variables: EnvironmentVariable;
+}
```

Comment:
We should keep a map of all environment and their variables. Since user will be switching between them, the local values will get cleared and will need to be set everytime

Reply:
```
export interface InitialState {
  currentEnvironment: EnvironmentDetails;
  envs: {
      [envName]: {
          variables: {
....
          }
      }
  }
}
```